
# Set the custom wrapper prefix for the entire workflow
snakemake.workflow.wrapper_prefix = "https://github.com/mmagdalenazz/snakemake-wrappers/raw/"



rule paraphase:
    input:
        bam="test_data/HG00733_smn1_realigned.bam",
        #fasta="test_data/homo_sapiens2.fa",
        fasta="test_data/chr5.fa",      
        #fasta="test_data/smn1_ref.fa",       
        vcf_header="test_data/vcf_chromosome_header.vcf",
    output:
        bam = "paraphrase/",
        merged_vcf = "paraphase/Sample1_paraphase.vcf.gz",
    params:
        genome="--genome 38",
        extra="-r test_data/chr5.fa -g smn1",
        outfolder=directory("paraphase/"),
    log:
        "paraphase/HG00733_smn1_realigned.vcf.log",
    threads: 1
    resources:
        mem_mb=1024,
    message:
        "{rule}: Calls paraphase on {input.bam} with paraphase to resolve SNVs in gene families"
    wrapper:
        #file
        "ppp/bio/paraphase"

